yaksh/models.py

```diff
@@ -1284,6 +1284,11 @@ def get_question_answers(self):
                 }]
         return q_a
 
+    def get_latest_answer(self,question_id):
+        all_answers = self.answers.filter(question=question_id)
+        if all_answers:
```

Comment:
This looks wrong, as I am not sure that all_answers[0] is indeed the latest answer, instead use `self.answers.latest('id')`?